core/edit/edit_command_modifiers.py

```diff
@@ -64,12 +64,21 @@ def edit_modifier(m) -> EditModifier:
 }
 
 
-def run_modifier_callback(modifier: EditModifier):
-    modifier_type = modifier.type
-    if modifier_type not in modifier_dictionary:
-        raise ValueError(f"Unknown edit modifier: {modifier_type}")
-
-    count = modifier.count
-    modifier = modifier_dictionary[modifier_type]
-    for i in range(1, count + 1):
-        modifier.callback()
+@mod.action_class
+class Actions:
+    def run_modifier_callback(modifier: EditModifier):
+        """
+        Run a callback that selects or prepares text ready to apply and edit action.
+        Intended for internal use and overwriting
+        """
+        count = modifier.count
+        modifier = actions.user.get_modifier_callback(modifier)
+        for i in range(1, count + 1):
+            modifier.callback()
```

Comment:
May be better to just have `get_modifier_callback()` return a callable? Otherwise any context specific implementation of this needs to construct the dataclass.